joomla/joomla-cms
# [4.0]Install default language fix

Pull Request for Issue # https://github.com/joomla/joomla-cms/issues/29896.

### Summary of Changes
Change FontAwesome Icons


### Testing Instructions
See https://github.com/joomla/joomla-cms/issues/29896. I copied instructions form there.


1. Install a clean 4.0-dev.
2. When you get to the first screen which lets you Install Additional Languages, click to install a new language.
3. Select French fr-FR or German de-DE and click Next button.
4. In the next screen choose a default admin and site language and click on Set default language 

-> No need for a chevron there, I removed it 
PLEASE TEST THIS

5. Then click again on Install Additional Languages button 

-> The chevron should be left and placed before the text in LTR, the opposite in RTL. I implemented this PLEASE TEST THIS in German and Persion
